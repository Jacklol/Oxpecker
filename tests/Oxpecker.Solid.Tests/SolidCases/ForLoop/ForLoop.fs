module Oxpecker.Solid.Tests.Cases.ForLoop

open Oxpecker.Solid

[<SolidComponent>]
let ForLoop () =
    div() {
        For(each = ["one"; "two"; "three"]) {
            fun (item: string) index ->
                h2(id = string index) {
                    item
                }
        }
    }
