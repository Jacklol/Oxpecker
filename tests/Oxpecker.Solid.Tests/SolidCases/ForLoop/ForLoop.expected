import { int32ToString } from "./fable_modules/fable-library-js.4.21.0/Util.js";

export function ForLoop() {
    return <div>
        <For each={["one", "two", "three"]}>
            {(item, index) => <h2 id={int32ToString(index)}>
                {item}
            </h2>}
        </For>
    </div>;
}

